zcash_test_vectors/orchard_zsa/commitments.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+import sys
+
+assert sys.version_info[0] >= 3, "Python 3 required."
+
+from ..orchard.group_hash import group_hash
+from ..orchard.pallas import Fp, Scalar, Point
+from ..orchard.sinsemilla import sinsemilla_hash_to_point
+from ..utils import i2lebsp, leos2bsp
+from .asset_base import zsa_value_base, asset_digest, encode_asset_id, native_asset
+
+# Commitment schemes used in Orchard https://zips.z.cash/protocol/nu5.pdf#concretecommit
+from ..orchard.commitments import rcv_trapdoor, L_ORCHARD_BASE
+
+def value_commit(rcv: Scalar, v: Scalar, asset: Point):
```

Comment:
Is this the same as https://github.com/QED-it/zcash-test-vectors/blob/zsa1/zcash_test_vectors/orchard/commitments.py#L18 ? If so, maybe we should just import.